app/src/sharedTest/java/org/oppia/android/app/onboarding/CreateProfileFragmentTest.kt

```diff
@@ -589,6 +594,34 @@ class CreateProfileFragmentTest {
     }
   }
 
+  @Test
+  fun testFragment_profileTypeArgumentMissing_showsUnknownProfileTypeError() {
+    val intent = CreateProfileActivity.createProfileActivityIntent(context)
+    intent.apply {
```

Comment:
Can we do this without ``apply``? Ditto below. I think it hurts the code readability as it's not obvious why ``apply`` is being used here.

Reply:
Done here, although below was a simplification of adding multiple params to the intent(modified now).